doc/building.md

```diff
@@ -42,7 +42,8 @@ Dependency list:
     CR    sdl2
     CR    sdl2_image
     CR    opusfile
-      A   opus-tools
+      A   libopus
+      A   libogg
```

Comment:
i think they are both `CRA`, because opus is needed at compile and runtime (via opusfile).